common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1723,6 +1728,23 @@ object CelebornConf extends Logging {
       .intConf
       .createOptional
 
+  val NETWORK_WILDCARD_ADDRESS_BIND: ConfigEntry[Boolean] =
+    buildConf("celeborn.network.bind.wildcardAddress")
+      .categories("network")
+      .version("0.6.0")
+      .doc("When `true`, the bind address will be set to a wildcard address, while the advertise address will " +
```

Comment:
Is "wildcard address" an idiomatic networking term used to represent an arbitrary address(`0.0.0.0` in IPv4 case and `::0` in IPv6)? I also saw [a similar question](https://github.com/apache/celeborn/pull/2633#discussion_r1690084955) in the previous PR.

I'm not sure what the official or most popular names of `0.0.0.0` and `::0` are, according to https://en.wikipedia.org/wiki/0.0.0.0, the most common one might be `INADDR_ANY`.

Reply:
I believe "wildcard address" with the current docs would confuse some users (including me), based on the results returned by the search engines (Google and Bing) "anyAddress" might be a better option. Anyway, please at least enrich the doc of this configuration.

Reply:
according to InetAddress docs it is defined as `wildcard address`: https://docs.oracle.com/javase/7/docs/api/java/net/InetSocketAddress.html#InetSocketAddress(int) 
I can make the doc more clear, thanks for bringing this up @pan3793

Reply:
okay, let's keep the name and update the doc then.